gatsby-node.js

```diff
@@ -1,12 +1,21 @@
+require('babel-register')
```

Comment:
curious as to why you're using `require('babel-regiester')` and not `import 'babel-register'`

Reply:
import doesn't work in this file because gatsby-node.js is not preprocessed by babel (that's why we need babe-register)